src/core/store/modules/lines/lines.reducer.ts

```diff
@@ -105,9 +109,21 @@ const linesReducer = createReducer(linesInitialState, (builder) => {
     })
 
     .addCase(networkChange, (state, { payload: { network } }) => {
+      // wallet action
       state.network = network;
     })
 
+    // synchronize network across network, lines, and wallet state
+    .addCase(changeNetwork.fulfilled, (state, { payload }) => {
```

Comment:
why is this in lines reducer and not network reducer?

Reply:
removed